lib/companion/companion_web/companion_web.ex

```diff
@@ -0,0 +1,123 @@
+defmodule Companion.CompanionWeb do
+  @moduledoc """
+  The CompanionWeb context.
+  """
```

Comment:
Should this be in `lib/companion/companion_web/companion_web.ex` or `lib/companion/companion_web.ex`?

I don't know how Phoenix apps are usually structured, but it seems pretty common in the Elixir world for `Foo.Bar` to be in `lib/foo/bar.ex` and `Foo.Bar.Baz` to be in `lib/foo/bar/baz.ex`.

Reply:
This was autogenerated, so I assumed that it was the correct place for a  phoenix app? I think maybe the reasoning is that there's also a `Companion.CompanionWeb.Application`, which should be placed at `lib/companion/companion_web/application.ex`, so maybe they're avoiding having both a folder and file named `companion_web`?

If you don't see any issues with having a folder and a file with the same name, then I'm happy to move it

Reply:
See `lib/`. :-)

There's no particular problem with it being where it is, so if that's where the autogenerator put it we should be fine leaving it where it is. (Elixir doesn't actually care about filenames or directory structure at all -- only the module definitions within the files. It's perfectly legal to put a `Foo.Bar.Baz` module in `lib/thing/not_at_all_foo/terrible_filename.ex`, but it's generally not a very good idea.)